plugins/polio/js/src/constants/messages.ts

```diff
@@ -2478,6 +2478,10 @@ const MESSAGES = defineMessages({
         id: 'iaso.polio.label.chronogram',
         defaultMessage: 'Chronogram',
     },
+    NotShowTestCampaigns: {
```

Comment:
```suggestion
    notShowTestCampaigns: {
```

Everything else is starting with a lowercase character.